tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -60,17 +61,135 @@
+# `S#_D# type` or `S#_D# wavelength`, where # is replaced
+# by the appropriate source and detector number, type is
+# either hbo or hbr, and wavelength is specified in nm.
+
+ch_names = ['D1_S1 hbo', 'D1_S1 hbr', 'D1_S2 hbo', 'D1_S2 hbr',
+            'D1_S3 hbo', 'D1_S3 hbr', 'D1_S4 hbo', 'D1_S4 hbr',
+            'D2_S5 hbo', 'D2_S5 hbr', 'D2_S6 hbo', 'D2_S6 hbr',
+            'D2_S7 hbo', 'D2_S7 hbr', 'D2_S8 hbo', 'D2_S8 hbr']
+ch_types = ['hbo', 'hbr', 'hbo', 'hbr',
+            'hbo', 'hbr', 'hbo', 'hbr',
+            'hbo', 'hbr', 'hbo', 'hbr',
+            'hbo', 'hbr', 'hbo', 'hbr']
+sfreq = 10.  # Hz
+
+
+###############################################################################
+# Finally, the data can be converted in to an MNE data structure.
+# The metadata above is used to create an :class:`mne.Info` data structure,
+# and this is combined with the data to create
+# an MNE :class:`~mne.io.Raw` object. For more details on the info structure
+# see :ref:`tut-info-class`, and for additional details on how continuous
+# data is stored in MNE see :ref:`tut-raw-class`.
+# For a more extensive description of how to create MNE data structures from
+# raw array data see :ref:`tut_creating_data_structures`.
+
+info = mne.create_info(ch_names=ch_names, ch_types=ch_types, sfreq=sfreq)
+raw = mne.io.RawArray(data, info, verbose=True)
+
+
+###############################################################################
+# Applying standard sensor locations to imported data
+# ---------------------------------------------------
+#
+# Having information about optode locations may assist in your analysis.
+# Beyond the general benefits this provides
+# (e.g. creating regions of interest, etc),
+# this is particularly important for fNIRS as information about the
+# distance between optodes is required to convert the optical density data
+# in to an estimate of the haemoglobin concentrations.
+# MNE provides methods to load standard sensor configurations (montages) from
+# some vendors, and this is demonstrated below.
+# Some handy tutorials for understanding sensor locations, coordinate systems,
+# and how to store and view this information in MNE are:
+# :ref:`tut-sensor-locations`, :ref:`plot_source_alignment`, and
+# :ref:`ex-eeg-scalp`.
+#
+# Below is an example of how to load the optode positions for an Artinis
+# OctaMon device. However, many fNIRS researchers use custom optode montages,
+# in this case you can generate your own .elc file (see `example file
+# <https://github.com/mne-tools/mne-python/blob/main/mne/channels/data
+# /montages/standard_1020.elc>`_) and load that instead.
+
+raw.set_montage('artinis-octamon')
+
+# View the position of optodes in 2D to confirm the positions are correct.
+raw.plot_sensors()
+
+
+###############################################################################
+# To validate the positions were loaded correctly it is also possible
+# to view the location of the sources (black), detectors (red),
```

Comment:
@rob-luke, the OctaMon has 1 detector in the middle surrounded by the 4 sources for each side, on the 3D plot we see red point in middle surrounded by black points, is that how it is supposed to be (red=detector and black=source) and do you have the same colour matching for NIRx?

Reply:
https://26372-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/io/plot_30_reading_fnirs_data.html#applying-standard-sensor-locations-to-imported-data

Reply:
Can you please check. Colors should be consistent.

Reply:
![pyvista](https://user-images.githubusercontent.com/32393169/111888493-e7cd5e00-89d4-11eb-9e35-c9375eb420e4.png)

I think that's okay actually because the NIRx has 8 sources and 16 detectors according to `ch_names`.

Reply:
In this picture red is a source. You can tell because the short detectors are black.